docs/software/module-usage.md

```diff
@@ -9,7 +9,7 @@ These all do the same thing:
 ```
 module available
 module avail
-module av
+module avail
```

Comment:
Is this duplicating line 11 now? Hard to tell in the github review screen!

Reply:
That is a duplicate, fixed